upcase_booklist_app/templates/users.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Curious Programmer | Users </title>
+        <link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+
+    <body>
+        <nav class="page-header">
+            <div class="header-items"><a href="{{ url_for('index') }}">Home</a></div> 
+            <div class="header-items"><a href="{{ url_for('book_list') }}">Books</a></div>
+            <div class="header-items"><a href="{{ url_for('authors') }}">Authors</a></div> 
+            <div class="header-items"><a href="{{ url_for('lists') }}">Lists</a></div>
+            <div class="header-items"><a href="{{ url_for('users') }}">Users</a></div>
+        </nav>
+
+        <div>
+            <h1>
+                Curious Programmer's Users
+            </h1>
+        </div>
+
+        <div>
+           
+            {% for user in users %}
```

Comment:
For any sort of list of objects, it's been to put them in an unordered list.

Reply:
However, this structure looks really good and you shouldn't have to do too much to add the `ul` and `li` elements.

Reply:
@paulghaddad I'm not sure where or why I'd add `ul` and `li` elements in this template? If anything, it may make sense to add these elements to the 'user.py' file?

```
<div class="user-details">
                    <p>{{ user_object['bio']}}</p>
                    <p>You can contact me at: {{ user_object['email'] }}</p>
                    <p>My favorite category is: {{ user_object['favorite_category']}}.</p>
                </div>
```

Reply:
It's a list of users...think if you just had a bulleted list of users, you'd use a `ul` and `li`s.